python/nano/src/bigdl/nano/pytorch/onnx/onnxrt_inference.py

```diff
@@ -208,5 +272,11 @@ def bind_onnxrt_methods(pl_model: LightningModule):
     pl_model.update_ortsess = partial(update_ortsess, pl_model)
     pl_model.on_fit_start = partial(on_fit_start, pl_model)
     pl_model.inference = partial(inference, pl_model)
+    pl_model.eval_onnx = partial(eval_onnx, pl_model)
+    pl_model._forward_onnx = partial(_forward_onnx, pl_model)
+    pl_model.exit_onnx = partial(exit_onnx, pl_model)
+    pl_model._torch_forward = pl_model.forward
+    pl_model._train_mode = pl_model.train
+    pl_model.train = partial(train, pl_model)
 
```

Comment:
for `_forward_args`, you may explicitly check if the model is `LightningModuleFromTorch`; if not, `pl_model.model` will be incorrect.